### ISSUE TITLE
SCK in use error on feather m4 express

### ISSUE THREAD

**Issue by jerryneedell** [adafruit/circuitpython#1088]
With the current master, if I execute the following script to mount my SD card 
```
import adafruit_sdcard
import busio
import digitalio
import board
import storage
import sys
# Connect to the card and mount the filesystem.
spi = busio.SPI(board.SCK, board.MOSI, board.MISO)
cs = digitalio.DigitalInOut(board.D5)
sdcard = adafruit_sdcard.SDCard(spi, cs)
vfs = storage.VfsFat(sdcard)
storage.mount(vfs, "/sd")
sys.path.append("/sd")
sys.path.append("/sd/lib")

```
 then try to execute a another script that attempts to use the SPI interface, I get an error

```
Adafruit CircuitPython 4.0.0-alpha-918-gdfa2581ff-dirty on 2018-08-03; Adafruit Feather M4 Express with samd51j19
>>> 
>>> 
>>> 
>>> import sdmount_lib
>>> import paint
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "paint.py", line 13, in <module>
ValueError: SCK in use
>>> 
```

If It run the same set of scripts on  a feather52832, they work without error.
If this error is correct, it would not be possible to execute a script that uses SPI from an SD Card.

For your reference, here is the paint.py script -- note, it is not stored on the SD Card in this example  -- on the feather52832, it is

```
"""
Simple painting demo that draws on an Adafruit capacitive touch shield with
ILI9341 display and STMPE610 resistive touch driver
"""

import busio
import board
import digitalio
import adafruit_stmpe610
from adafruit_rgb_display import ili9341, color565

# Create library object using our SPI port
spi = busio.SPI(clock=board.SCK, MOSI=board.MOSI, MISO=board.MISO)

# Adafruit Metro M0 + 2.8" Capacitive touch shield
cs_pin = digitalio.DigitalInOut(board.D9)
dc_pin = digitalio.DigitalInOut(board.D10)

# Initialize display
display = ili9341.ILI9341(spi, cs=cs_pin, dc=dc_pin)
# Fill with black!
display.fill(color565(0, 0, 0))

st_cs_pin = digitalio.DigitalInOut(board.D6)
st = adafruit_stmpe610.Adafruit_STMPE610_SPI(spi,st_cs_pin,baudrate=1000000)

while True:
    if st.touched:
        while not st.buffer_empty:
            ts = st.touches
            for point in ts:
                # perform transformation to get into display coordinate system!
                y = point['y']
                x = 4096 - point['x']
                x = 2 * x // 30
                y = 8 * y // 90
                display.fill_rectangle(x-2, y-2, 4, 4, color565(255, 0, 0))

```

**Comment by jerryneedell** [comment]
simple example to isolate issue

on feather_m4_express
```
Adafruit CircuitPython 4.0.0-alpha-918-gdfa2581ff-dirty on 2018-08-03; Adafruit Feather M4 Express with samd51j19
>>>
>>>
>>> import busio
>>> import board
>>> spi1 = busio.SPI(board.SCK, board.MOSI, board.MISO)
>>> spi2 = busio.SPI(board.SCK, board.MOSI, board.MISO)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: SCK in use
>>>
``` 

on on feather feather52832
``` 
Adafruit CircuitPython 4.0.0-alpha-918-gdfa2581ff-dirty on 2018-08-04; Bluefruit nRF52 Feather with nRF52832
>>> 
>>> 
>>> import busio
>>> import board
>>> spi1 = busio.SPI(board.SCK,board.MOSI,board.MISO)
>>> spi2 = busio.SPI(board.SCK,board.MOSI,board.MISO)
>>> 
```

**Comment by dhalbert** [comment]
Closed in favor of #1089.
